x/rollapp/keeper/latest_state_info_index.go

```diff
@@ -36,6 +36,17 @@ func (k Keeper) GetLatestStateInfoIndex(
 	return val, true
 }
 
+func (k Keeper) GetLatestHeight(
+	ctx sdk.Context,
+	rollappId string,
+) (uint64, bool) {
+	info, ok := k.GetLatestStateInfo(ctx, rollappId)
+	if !ok {
+		return 0, false
+	}
+	return info.GetLatestHeight(), true
```

Comment:
is this guaranteed latest height > 0?

Reply:
why does it matter? If there is a state info, then the rollap definitely updated the hub

Reply:
then it's good